docs/swupd.1.rst

```diff
@@ -168,7 +168,7 @@ SUBCOMMANDS
         are available. It will return 0 with succeeded and a different value
         of 0 with failed.
 
-    - `-d, --deps={bundle}`
+    - `-i, --deps={bundle}`
```

Comment:
This option isn't going to be used very frequent, so I would remove the short option

Reply:
I'll do that then

Reply:
btw, I had chosen "i" because the "deps" are actually the **i**ncludes

Reply:
if we are getting rid of the short option for --deps, we might as well get rid of the short option for --has-dep, don't you think? both are very similar so it would look odd to have a short option for one but not have it for the other. What do you think @otaviobp ?

Reply:
hnaahah. I didn't get that. 
I don't think --has-deps should have a short option, but I wouldn't remove it to avoid breaking compatibility. --deps breaks it, but it's needed. Let's try to avoid at lest adding more short options for all flags

Reply:
😩 ok, that kind of visually unattractive things bother me, but only because I'm very anal haha, so ok, I will just remove one short option.